packages/web-api-client/src/a11y-service-client.ts

```diff
@@ -84,7 +78,7 @@ export class A11yServiceClient {
         const requestUrl: string = `${this.requestBaseUrl}/scans/${scanId}/reports/${reportId}`;
 
         return (await this.retryHelper.executeWithRetries(
-            async () => (await this.signRequest()).get(requestUrl),
+            async () => (await this.signRequest()).get(requestUrl, { responseType: 'text' }),
```

Comment:
text/html is probably more accurate

Reply:
This is to tell got how to parse the response body, it only accepts 'json', 'text', or 'buffer.'